.github/workflows/backend.yml

```diff
@@ -73,30 +73,32 @@ jobs:
         /work/enterprise/backend/test
         /work/shared/src
         /work/shared/test
-        /work/modules/drivers/sparksql/src
-        /work/modules/drivers/sparksql/test
-        /work/modules/drivers/vertica/src
-        /work/modules/drivers/vertica/test
+        /work/modules/drivers/athena/src
+        /work/modules/drivers/athena/test
```

Comment:
We were actually not running Kondo against Athena in CI, so I added it and fixed a couple of errors. I sorted these to make then easier to read